aip/general/0127.md

```diff
@@ -92,6 +92,17 @@ message CreateBookRequest {
   - The `body` **must not** be a `repeated` field.
   - Fields **should not** use the `json_name` annotation to alter the field
     name in JSON, unless doing so for backwards-compatibility reasons.
+  - In any given HTTP request, if a field is represented in both the body
+    and the URI, the values **must** be identical. The server behavior when
+    this constraint is violated is unspecified.
+- Eligible fields which are not represented in the URI path or the body are mapped
+  to URI query parameters.
```

Comment:
It might be useful to include a reference to https://developers.google.com/protocol-buffers/docs/proto3#json, which has more details (and may get updated based on our parallel work)

Reply:
That's already referenced earlier in the body part: "The request body is encoded as JSON
  as defined by protocol buffers' canonical JSON encoding" where "JSON encoding" is a link.